.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 88
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
+ignore = E501
```

Comment:
I don't think it's necessary to ignore E501 as I wouldn't want to have any lines extend the max-line-length.

Reply:
No problem. Also, I would suggest using a formatter (black, yapf, pep8, etc) with the project to help maintain line-length and also improved readability for some of your more dense blocks (particularly in the session.py)